### DISCUSSION TITLE
What's Font did notice.nvim readme image use?

### DISCUSSION THREAD

**Discussion by ghulryk57656** [folke/noice.nvim#415] [Q&A] [answered]
Does Anyone know what font readme image use?
![image](https://user-images.githubusercontent.com/122608841/228132522-4d77f879-2d60-4af8-810a-33694928f203.png)

**Comment by yutkat** [comment, marked as answer]
I think it is Victor Mono.
https://rubjo.github.io/victor-mono/

**Reply by ghulryk57656** [reply]
Thanks!!!

**Comment by folke** [comment]
https://github.com/folke/dot/blob/master/config/kitty/kitty.conf

**Reply by ghulryk57656** [reply]
Thank you!!
